### PR TITLE
feat(ci): should build cryostat3 as well as cryostat (2.x)

### PR DESCRIPTION
**Pull request by aali309**
Fixes: #324

### CODE REVIEW

**Comment by aali309** [comment]
the build for cryostat3 works well. 
image

The failed test for `cryostat build`
image

**Comment by andrewazores** [comment]
Hmm... that 2.5.0 build should work since https://github.com/cryostatio/cryostat/commit/8ae79bc267ae5684a7fe539c4e92a01c784be1eb .

**Comment by aali309** [comment]
It works here , that was on my local branch.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -101,6 +106,72 @@ jobs:
...
+        mkdir -p $HOME/.bin
+        cat <(echo '#!/usr/bin/env bash') <(echo 'exec podman "$@"') > $HOME/.bin/docker
+        chmod +x $HOME/.bin/docker
+        echo "PATH=$HOME/.bin:$PATH" >> "$GITHUB_ENV"
+    - name: Start Podman API
+      run: systemctl --user enable --now podman.socket
+    - name: Set DOCKER_HOST environment variable
+      run: echo "DOCKER_HOST=unix:///run/user/$(id -u)/podman/podman.sock" >> "$GITHUB_ENV"
+    - name: Build cryostat3
+      run: ./mvnw -B -U -Dbuild.java=${{ matrix.java}}-Dio.cryostat.core.version=${{ needs.build-core.outputs.core_version }} clean package
```
Comment (andrewazores) [inline .github/workflows/ci.yml:172]: `-Dbuild.java=${{ matrix.java}}-Dio.cryostat.core.version` I'm surprised this works - looks like there's a missing space at the second `-D`?

I don't think it matters but `${{ matrix.java}}` has a missing space (`java }}`) too.

Comment (aali309) [reply .github/workflows/ci.yml:172]: yes! I see that.

**Comment by aali309** [comment]
image

**Comment by andrewazores** [comment]
I think either now or at some point in the near future, we will also want to update that ^ test image matrix to provide 3.0 test images, rather than 2.xs.

**Review by andrewazores — APPROVED** [review]

**Pull request merged**